extensions/rvdatabinding/src/main/java/com/google/android/agera/rvdatabinding/DataBindingRepositoryPresenterCompilerStates.java

```diff
@@ -65,6 +68,21 @@
     DBRPMain<T> onRecycle(@RecycleConfig int recycleConfig);
 
     /**
+     * Specifies a data binding @{code itemId} from the previously given {@code layout} to bind a
+     * single item in the {@link Repository}.
```

Comment:
`@{code` -> `{@code` (multiple instances; elsewhere in this file too)

... to bind the whole collection (the repository value) to.